sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_daily_v1/query.sql

```diff
@@ -17,41 +20,44 @@ CREATE TEMP FUNCTION udf_contains_tier1_country(
   )
 );
 
-  --
-  -- This UDF is also only applicable in the context of this query.
-CREATE TEMP FUNCTION udf_contains_registration(
-  x ANY TYPE
-) AS ( --
-  EXISTS(
-    SELECT
-      event_type
-    FROM
-      UNNEST(x) AS event_type
-    WHERE
-      event_type IN ( --
-        'fxa_reg - complete'
-      )
-  )
-);
-
-WITH base AS (
+WITH fxa_events AS (
   SELECT
     `timestamp`,
-    event_type,
     user_id,
-    app_version,
+    -- cert_signed is specific to sync, but these events do not have the
+    -- 'service' field populated, so we fill in the service name for this special case.
+    IF(
+      `service` IS NULL
+      AND event_type = 'fxa_activity - cert_signed',
+      'sync',
+      `service`
+    ) AS `service`,
     os_name,
     os_version,
+    app_version,
+    flow_id,
+    entrypoint,
+    event_type,
     country,
     `language`,
-    -- cert_signed is specific to sync, but these events do not have the
-    -- 'service' field populated, so we fill in the service name for this special case.
-    IF(service IS NULL AND event_type = 'fxa_activity - cert_signed', 'sync', service) AS service,
+    ua_version,
+    ua_browser,
+    utm_term,
+    utm_medium,
+    utm_source,
+    utm_campaign,
+    utm_content,
   FROM
     `moz-fx-data-shared-prod.firefox_accounts.fxa_all_events`
   WHERE
-    event_category IN ('content', 'auth', 'oauth')
-    AND user_id IS NOT NULL
+    DATE(`timestamp`)
+    -- 2 day time window used to make sure we can get user session attribution information
+    -- which will not always be available in the same partition as active user activity
+    -- ('fxa_login - complete', 'fxa_reg - complete').
+    -- this includes fields such as entrypoint, utm's etc.
+    BETWEEN DATE_SUB(@submission_date, INTERVAL 1 DAY)
+    AND @submission_date
```

Comment:
I think ```DATE(`timestamp`)``` is on the wrong side of this comment
```suggestion
    -- 2 day time window used to make sure we can get user session attribution information
    -- which will not always be available in the same partition as active user activity
    -- ('fxa_login - complete', 'fxa_reg - complete').
    -- this includes fields such as entrypoint, utm's etc.
    DATE(`timestamp`)
    BETWEEN DATE_SUB(@submission_date, INTERVAL 1 DAY)
    AND @submission_date
```